Open Judge System/Web/OJS.Web/Areas/Contests/Views/Results/_FullResultsPagedList.cshtml

```diff
@@ -64,12 +64,25 @@
                         <td style="white-space: nowrap">
                         @if (problemResult != null)
                         {
-                            <strong><a href="@(Url.Action<SubmissionsController>(c => c.Details(problemResult.BestSubmission.Id)))">@problemResult.BestSubmission.Points / @problemResult.MaximumPoints</a></strong>
-                            @problemResult.BestSubmission.SubmissionType
+                            if (problemResult.BestSubmission.Id.HasValue)
+                            {
+                                <strong><a href="@(Url.Action<SubmissionsController>(c => c.Details(problemResult.BestSubmission.Id ?? default(int))))">@problemResult.BestSubmission.Points / @problemResult.MaximumPoints</a></strong>
```

Comment:
Consider putting `a` tag on separate line or moving the `href` value in separate variable. As of now the line seems a bit long.